ui_test/src/lib.rs

```diff
@@ -75,10 +76,10 @@ pub fn run_tests(mut config: Config) -> Result<()> {
 
     let dependencies = build_dependencies(&config)?;
     for (name, dependency) in dependencies {
-        config.args.push("--extern".to_string());
-        config.args.push(format!("{name}={}", dependency.display()));
-        config.args.push("-L".to_string());
-        config.args.push(dependency.parent().unwrap().display().to_string());
+        config.args.push("--extern".into());
+        config.args.push(format!("{name}={}", dependency.display()).into());
+        config.args.push("-L".into());
+        config.args.push(dependency.parent().unwrap().display().to_string().into());
```

Comment:
`dependency` is a `Path`, right? So you are converting that to a String and then back to OsString...?
Just use `into_os_string`. :)

Reply:
I already did that in a follow up commit